doc/rados/operations/control.rst

```diff
@@ -243,18 +243,23 @@ Deployments utilizing Nautilus (or later revisions of Luminous and Mimic)
 that have no pre-Luminous cients may instead wish to instead enable the
 `balancer`` module for ``ceph-mgr``.
 
-Add/remove an IP address to/from the blocklist. When adding an address,
+Add/remove an IP address or CIDR range to/from the blocklist.
+When adding to the blocklist,
 you can specify how long it should be blocklisted in seconds; otherwise,
 it will default to 1 hour. A blocklisted address is prevented from
-connecting to any OSD. Blocklisting is most often used to prevent a
-lagging metadata server from making bad changes to data on the OSDs.
+connecting to any OSD. If you blocklist an IP or range containing an OSD, be aware
+that OSD will also be prevented from performing operations on its peers where it
+acts as a client. (This includes tiering and copy-from functionality.)
```

Comment:
Should we explicitly mention backfill, recovery, replication?  Are those subsumed by "copy-from"?

Reply:
The OSD is not a client to other OSDs when doing backfill and other peering ops. We can update that but I'd rather do it against main and then backport the doc update.